rust/libceed/src/vector.rs

```diff
@@ -273,15 +273,15 @@ impl<'a> Vector<'a> {
     ///
     /// # arguments
     ///
-    /// * `vec_copy` - vector to copy array values to
+    /// * `vec_source` - vector to copy array values from
     #[allow(unused_mut)]
-    fn vec_copy_mut(
+    fn copy_from(
         mut self,
-        vec_copy: crate::Vector,
+        vec_source: &crate::Vector,
```

Comment:
I think I missed below, return type of `crate::Result<Self>` now, like with `from_slice`?

Reply:
ah, that makes sense.